src/stcal/jump/twopoint_difference.py

```diff
@@ -119,25 +117,41 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
     dnu_flag = dqflags["DO_NOT_USE"]
     jump_flag = dqflags["JUMP_DET"]
 
+    # get data, gdq
+    dat = dataa
```

Comment:
Just curious as the purpose of `dat`?  It's just a view into the existing `dataa`, so why can't `dataa` just be used? If you want a separate copy, in order to prevent anything from getting changed in `dataa`, then you need to explicitly make `dat` a copy, rather than just another view of `dataa`.

Reply:
This is left over from someone else's changes (I think). I was never clear on why it was done that way.
I agree that it doesn't make sense. I think it had something to do with the copying but that's done earlier and doesn't need to be done again.
I made changes at the if test on line 101 that makes more sense.